cypress/integration/dateRange.feature

```diff
@@ -0,0 +1,76 @@
+Feature: Date Range component
+  I want to change Confirm component properties
+
+  Background: Open Date Range component page
+    Given I open "Experimental Date Range" component page
+
+  @positive
+  Scenario Outline: Change Date Range start label
+    When I set startLabel to "<label>"
+    Then start label on preview is "<label>"
+    Examples:
+      | label                    |
+      | Sample text              |
+      | 1234567890               |
+      | áéíóú¿¡üñ                |
+      | !@#$%^*()_+-=~[];:.,?{}  |
+      | ÄÖÜßäöüß                 |
+      | <>                       |
+
+  @positive
+  Scenario Outline: Change Date Range end label
+    When I set endLabel to "<label>"
+    Then end label on preview is "<label>"
```

Comment:
please make `end label` consistant and replace with `endLabel`

Reply:
fixed